update-billing.rst

```diff
@@ -0,0 +1,13 @@
+.. _how-to-update-billing:
+
+How to update credit card and/or billing information
+=====================================================
+
+To update your credit card and/or your billing information, go to the *organisation seetings* of your project and select
```

Comment:
Spelling: "settings".

Reply:
Done. How did I press seeting instead of setting more than once, strange!